front/app/modules/commercial/id_vienna_saml/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { ModuleConfiguration } from 'utils/moduleUtils';
+import ViennaSamlButton from './components/ViennaSamlButton';
+import FeatureFlag from 'components/FeatureFlag';
+
+const configuration: ModuleConfiguration = {
+  outlets: {
+    'app.components.SignUpIn.AuthProviders.ContainerEnd': (props) => {
+      return (
+        <FeatureFlag name="vienna_citizen_login">
```

Comment:
Not a big deal but it's preferred to use `useFeatureFlag` inside of the component in question (in this case `ViennaSamlButton`)

Reply:
What is the advantage here? Can do!